<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Zeugs Control Panel</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Current Database</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_db" >
         <property name="minimumSize" >
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Host</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_host" >
           <property name="minimumSize" >
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="mouseTracking" >
            <bool>false</bool>
           </property>
           <property name="focusPolicy" >
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="acceptDrops" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="label_finalized" >
           <property name="minimumSize" >
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>Active</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Quit" >
           <property name="text" >
            <string>Quit</string>
           </property>
           <property name="icon" >
            <iconset>icons/quit.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_5" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_db" >
      <attribute name="title" >
       <string>database</string>
      </attribute>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <widget class="QPushButton" name="pushButton_updatedb" >
           <property name="text" >
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QPushButton" name="pushButton_finalize" >
           <property name="text" >
            <string>Finalized</string>
           </property>
           <property name="checkable" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QPushButton" name="pushButton_dump" >
           <property name="text" >
            <string>Backup</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QPushButton" name="pushButton_print" >
           <property name="text" >
            <string>Print</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QPushButton" name="pushButton_newdb" >
           <property name="text" >
            <string>New</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QPushButton" name="pushButton_sync" >
           <property name="toolTip" >
            <string>Synchronize with user file</string>
           </property>
           <property name="text" >
            <string>Synchronize</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Extra</string>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_restore" >
            <property name="text" >
             <string>Restore from backup</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_dumpd" >
            <property name="text" >
             <string>Dump config files</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_dbdel" >
            <property name="text" >
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_teachers" >
      <attribute name="title" >
       <string>teachers</string>
      </attribute>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QListWidget" name="listWidget_users" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButton_gen" >
               <property name="text" >
                <string>Generate user databases</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_pwd" >
               <property name="text" >
                <string>Reset password</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="0" column="0" >
            <widget class="QPushButton" name="pushButton_usel" >
             <property name="text" >
              <string>Select all</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QPushButton" name="pushButton_uinvsel" >
             <property name="text" >
              <string>Invert selection</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QPushButton" name="pushButton_uunsel" >
             <property name="text" >
              <string>Unselect all</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Quit</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>385</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel" >
     <x>429</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
